hubmap/frontend/src/App.css

```diff
@@ -121,11 +121,11 @@ background-color: #fafafa;
   margin-top: 100px;
 }
 
-.studieschart{
+.experimentschart{
 
 }
 
-.studiesdashboard{
+.experimentsdashboard{
```

Comment:
Can we punctuate multi-word classes? I would favor `experiments-dashboard`.

Reply:
can we use hyphenated class names in react ?

Reply:
Unless I'm missing something, it's only used here:
```
                <div className="experimentsdashboard" id="experimentsbyTissueschart" >
```
The react docs for `className` use a dash: https://reactjs.org/docs/faq-styling.html

(FWIW: I always prefer using dashes in CSS class names and IDs: I think one reason folks do this is that since other contexts don't allow dashes, it provides a kind of name-spacing. For instance, if you're searching for usages, you'll find only the CSS class names, and not get similar names from JS or Python.)

(... and in that line above, every word in the ID should be marked, whatever convention you use... but in this case, I can't find any references to the ID, so it shouldn't even be there?)